langserver/workspace_refs.go

```diff
@@ -33,8 +36,23 @@ func (h *LangHandler) handleWorkspaceReference(ctx context.Context, conn JSONRPC
 		pkgPat = h.init.RootImportPath + "/..."
 	}
 
+	var paralellism int
```

Comment:
_Shoooot._ That word is so hard to spell. Fixed in f8788d94052a4cbc8b5bd6cf590c451bb6e1d80b